### PR TITLE
App plugin cleanup

### PR DESCRIPTION
**Pull request by bradenmacdonald**
This PR removes a bunch of cruft from the repo that doesn't really apply to an Open edX plugin app.

Test instructions:
* Clone to your devstack's `src` directory and checkout this branch
* Install: `make studio-shell`, `pip install -e /edx/src/openedx-blockstore-relay/`
* Test (in studio-shell): `make -f /edx/src/openedx-blockstore-relay/Makefile validate`

Changes:
* I removed most of the requirements specifications because as a plugin, this cannot use different requirements from what Open edX is already using.
* I simplified the Makefile, removing some targets we don't use and making it work from another directory like `edx-platform`
* I made some quality fixes to get pylint etc. passing.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,37 +43,45 @@ docs: ## generate Sphinx HTML documentation, including API docs
...
+test: clean
+	python -m pytest --ds=cms.envs.test --no-cov --nomigrations $(PROJECT_ROOT)openedx_blockstore_relay/tests/
 
 diff_cover: test ## find diff lines that need test coverage
 	diff-cover coverage.xml
 
-test-all: ## run tests on every supported Python/Django combination
-	tox -e quality
-	tox
+# test-all: ## run tests on every supported Python/Django combination
```
Comment (symbolist) [inline Makefile]: Do we want to remove the commented-out files?

**Review by symbolist — APPROVED** [review]
I do see this warning when running validate.

```
pycodestyle /edx/src/openedx-blockstore-relay/openedx_blockstore_relay /edx/src/openedx-blockstore-relay/setup.py
isort --check-only --diff --recursive openedx_blockstore_relay /edx/src/openedx-blockstore-relay/setup.py
WARNING: Unable to parse file openedx_blockstore_relay due to [Errno 2] No such file or directory: '/edx/app/edxapp/edx-platform/openedx_blockstore_relay'
```
But besides it, everything else looks 👍 

- [x] I tested this by following the testing steps
- [x] I read through the code
- [x] I checked for accessibility issues
- [x] Includes documentation

File: Makefile (hunk 1 above)
Comment (bradenmacdonald) [reply Makefile]: I have to keep `tox.ini` because it contains the `pycodestyle` settings etc. I can remove these few lines from the makefile though.

**Comment by bradenmacdonald** [comment]
Thanks @symbolist ! I fixed `isort` and confirmed the tests are passing again, so now I'll merge.

**Pull request merged**